app/security/Roles.scala

```diff
@@ -183,7 +183,7 @@ object Roles {
   val userJourneySequence: List[(CsrAuthorization, Call)] = List(
     ApplicationStartRole -> routes.HomeController.present,
     PersonalDetailsRole -> routes.PersonalDetailsController.present(None),
-    SchemesRole -> routes.SchemeController.entryPoint,
+    SchemesRole -> routes.SchemePreferencesController.present,
```

Comment:
Was @ollyw just talking about standardising this? I'm not sure this method is declared with empty parentheses